### ISSUE TITLE
Tables on Data Portal should have row and column HTML headers

### ISSUE THREAD

**Issue by adriennes** [DataBiosphere/data-portal#391]
All tables on the Data Portal should have 'row' and 'column' headers in HTML to make them accessible by screen readers.  
Specific issues: 
- Data Browser should be treated like one big table, with each column given a header and each item in the list treated as a row. 
- Learn > Introduction > Data Processing Pipelines> What is the data processing pipeline service? - Table should have row and column headers. 
- Learn > Introduction > Data Processing Pipelines> Smart-seq2 Workflow - Table should have row and column headers. 
- Learn > Introduction > Data Processing Pipelines> QC Metrics - Table should have row and column headers. 
- Learn > Metadata Dictionary - Should be treated as a table, with row and column headers.
